src/CoreGraphics/CGPath.cs

```diff
@@ -560,7 +560,7 @@ public void Apply (ApplierFunction func)
 		static CGPath MakeMutable (IntPtr source)
 		{
 			var mutable = CGPathCreateMutableCopy (source);
-			return new CGPath (mutable, false);
+			return new CGPath (mutable, true);
```

Comment:
Let's start using the parameter name whenever we add/modify this pattern, i.e.
`return new CGPath (mutable, own: true);`
because the negation inside the ctor to retain is just not something we're able to review correctly 100% of the time. After years I still have to fight not to see this as `retain: false` :(